### ISSUE TITLE
Batch send to 4 addresses hangs the app, can recreate

### ISSUE THREAD

**Issue by tattwamasi** [spesmilo/electrum#6897]
I have 4 addresses (from a csv file, but can paste them in too, same effect) that will cause an app hang if you hit the pay... button on MacOS.  Soon as you hit the pay button you get the spinning beach ball.  The next window where you set your fees, etc. never pops up.

The odd thing is, if I remove any one or more addresses it works, but as long as I have all 4 there, whatever order, it hangs.  Have done many batch sends of various sizes, much larger, though none this small.  Haven't seen this behavior before.

This is on version 4.0.9. I can provide the csv if this isn't a duplicate issue and it's useful.

**Comment by ecdsa** [comment]
please paste the addresses here.
also check the logs of your electrum, to see if there is anything useful

**Comment by tattwamasi** [comment]
First I must correct my report.  It's just one bad address - I missed a test step.  
The address is bc1qzgf52caexw68zlz0h5972lvqykhxyzp3exvrvu
It doesn't matter what amount is tried.  I can't see a log in the UI as it hangs soon as you hit pay...
Is there a file log?

**Comment by ecdsa** [comment]
yes, there are file logs in your electrum directory. 
the address is fine, the issue does not come from that.

**Comment by tattwamasi** [comment]
which directory?  .electrum?  Maybe you have to first select the write logs to file checkbox in general settings.  Trying again with that checked and seeing.  
The address is the only thing that triggered it.  I did the other three fine.  Looking for logs...

**Comment by ecdsa** [comment]
I suppose the issue is not with the address, but with the amount you are trying to send to that address.
try to send the same amount to another address, for example.

**Comment by tattwamasi** [comment]
I had tried different amounts with the address.  Didn't matter.  But perhaps it's moot and was a passing problem...
After enabling logs, the pay window popped up just as expected.  After being stunned that this start it worked after repeated force quits, I tried it again and it worked again.  I disabled logging and tried it again out of curiosity, and it worked correctly then too.

I have no explanation.
